bridges/TwitterV2Bridge.php

```diff
@@ -70,6 +70,11 @@ class TwitterV2Bridge extends BridgeAbstract
                 'type' => 'checkbox',
                 'title' => 'Activate to display original sized images (no thumbnails)'
             ],
+            'noexternallink' => [
+                'name' => 'Hide externel link from content html',
+                'type' => 'checkbox',
+                'title' => 'Activate to hide images in tweets'
```

Comment:
This is not an accurate description of what this option does

Reply:
```suggestion
                'title' => 'Activate to hide the links from the content html field'
```